Sample_Data/TSU/TSUSample_DigCollYearbooksCatalogues.xml

```diff
@@ -0,0 +1,582 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I really hate putting XML like this in the repository.  It makes the repo unmanageable and makes it so you can't actually use GitHub functions.  I know this is a thing that has been done in the past, but I really think this is unnecessary.  Technically, Repox doesn't even run against XML like this so it's not even a true way to test.

I will approve and merge with this here, but I still feel strongly that this should be considered.

Reply:
To clarify, are you suggesting that we shouldn't include any XML in this repository? What I've added is a small sample of what is present mostly for testing purposes. I'm not sure where the cut off is in terms of length where GitHub functions no longer runs. I'm happy to remove this and just add XML as a text file.

Reply:
This helps me understand - I thought you just didn't want huge XML files.